roles/contiv_network/tasks/ovs.yml

```diff
@@ -53,9 +53,16 @@
     - "tcp:127.0.0.1:6640"
     - "ptcp:6640"
 
+- name: check selinux status
+  shell: sestatus | egrep "SELinux status:.*enabled"
+  register: selinux_set
+  when: (ansible_os_family == "RedHat")
```

Comment:
Looks good to me. But I need to hear from @vvb about ubuntu issue. I am pretty sure he ran in to that issue and even I was facing that issue so that was whole need to this PR.

@vvb : Could you please comment on following things.

1: on ubuntu, Are you able to execute net_demo_installer script correctly or not ?
2: On ReadHat / CentOs:
   2.1: Check net_demo_installer script when Selinux is disabled.
   2.2: Check net_demo_installer script when Selinux is enabled and is in enforcing mode
   2.3 : Check net_demo_installer script when Selinux is enabled and is in Permissive mode.

I dont have ubuntu setup. If you have please point me the details and I will be happy to test this.